$(top_builddir)/jabberd/gcrypt_init.c is using function gcry_control() so we need to link against -lgcrypt

http://bugs.gentoo.org/412871

--- jabberd/Makefile.am
+++ jabberd/Makefile.am
@@ -20,5 +20,5 @@
 lib_LTLIBRARIES = libjabberd.la
 
 libjabberd_la_SOURCES = acl.cc config.cc gcrypt_init.c heartbeat.cc mio.cc mio_tls.cc mtq.cc xdb.cc deliver.cc log.cc mio_raw.cc mio_xml.cc subjectAltName_asn1_tab.c
-libjabberd_la_LIBADD = -lexpat $(top_builddir)/jabberd/lib/libjabberdlib.la
+libjabberd_la_LIBADD = -lexpat -lgcrypt $(top_builddir)/jabberd/lib/libjabberdlib.la
 libjabberd_la_LDFLAGS = @LDFLAGS@ @VERSION_INFO@ -export-dynamic -version-info 2:0:0
